### PR TITLE
Hotfix/change user profile picture

### PR DESCRIPTION
**Pull request by kzepfl**
## Changes

- Changed in `Profile.kt` how User profile picture is displayed
_User picture now use the same function as friend picture to display their image
- Fixe in `SignIn.kt`. Now when a new user is created, it will fetch their profile picture from Firebase and Update profile picture during signin
- Added a function `updateUserPicture` in user MVVM.

### CODE REVIEW

**Comment by kzepfl** [comment]
After some test it seems like the google url doesnt change even after you change your profile picture, there might just be some delay where you have your oléd picture instead of your current.
## Changes

- deleted all the function of user mvvm for updating the user profile picture

File: app/src/test/java/com/android/streetworkapp/ui/authentication/SignInTest.kt [hunk 1]
```diff
@@ -16,16 +16,16 @@ class SignInTest {
...
   @Test
   fun testCreateUserFromFirebaseUser() {
     // Mock the FirebaseUser properties
     Mockito.`when`(firebaseUser.uid).thenReturn("testUid")
-    Mockito.`when`(firebaseUser.displayName).thenReturn("Test User")
-    Mockito.`when`(firebaseUser.email).thenReturn("testuser@example.com")
 
-    // Call the method to test
+    // Call the method to testb
     val user = createNewUserFromFirebaseUser(firebaseUser)
```
Comment (Simmanz) [inline app/src/test/java/com/android/streetworkapp/ui/authentication/SignInTest.kt:29]: I think this is a typo :)

**Review by Simmanz — CHANGES_REQUESTED** [review]
Good PR ! It is great that you abstract away the pictures in a common component ! Only two minor things though, could you fix the picture **padding** and the `testb` typo ?

<p align="center">
image
</p>

**Review by Simmanz — APPROVED** [review]
All good now ! The typo has been fixed and the padding is great :

<p align="center">
image
</p>

**Pull request merged**